types/connect-mongo/tsconfig.json

```diff
@@ -7,6 +7,9 @@
         "paths": {
             "mongodb": [
                 "mongodb/v2"
+            ],
+            "mongoose": [
+                "mongoose/v4"
```

Comment:
It would be better to use the latest mongoose types if these libraries are compatible -- were there compile errors without this?

Reply:
I was just updating all the dependent packages as per README instructions. Will revert.

Reply:
de78ab2174ad552bc26ea0f2d120e3462963776a

Reply:
I'll update the README -- it's only necessary to do this if there would otherwise be compile errors.

Reply:
#23223